x/auth/ante/unordered.go

```diff
@@ -45,11 +45,13 @@ func (d *UnorderedTxDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate b
 		return next(ctx, tx, simulate)
 	}
 
-	if unorderedTx.GetTimeoutHeight() == 0 {
+	// TTL is defined as a specific block height at which this tx is no longer valid
+	ttl := unorderedTx.GetTimeoutHeight()
+
+	if ttl == 0 {
```

Comment:
ttl < ctx.BlockHeight()

?

Reply:
Yeah, I'll add that check too to be safe. I didn't add it originally because the original timeout decorator checked this.